Lib/email/utils.py

```diff
@@ -215,6 +215,12 @@ def parsedate_to_datetime(data):
 
 
 def parseaddr(addr):
+    """
+    Parse addr into its constituent realname and email address parts.
+    
+    Return a tuple of realname and email address, unless the parse fails, in which
```

Comment:
Oops, I just noticed that 'which' goes beyond 79 columns.  (Too bad github doesn't make that obvious.)  So you need to move that word to the next line.

Reply:
I thought that it was just one character short of 79. I've made the changes